.travis.yml

```diff
@@ -0,0 +1,80 @@
+# Copyright Contributors to the Open Cluster Management project
+
+os:
+  - linux
+
+language: go
+go:
+  - 1.15.x
+
+services:
+  - docker
+
+addons:
+  sonarcloud:
+    organization: "open-cluster-management"
+    token:
+      secure: "Mkhnn3nG95bDBBHZaRn1VY6QX+q5IRlDyvtRFZhDhfQVulF5VTboY75uJCifQl093OfyPGJOarwK+2BQbhlfsZuGJMRHICc5qqPpj4SUWMIq8jRSr+zUEqrqgR2Q1QSb8XFPJl9v2WUPg4PyUY7GfgFAOi40rkyvarphjCQb6EIVGNvPHXIh6N+hopZ6mlnE3H7vzHBT9yE9DWS8NKg7KncXTNF/CoBwoWxL24V+aSMSknRpUE6EZRGjbyZzxYf8De3dnoNVlB21e1UMtzc0owG3xYxNsDgv7Las7FXM2LRy50uX7/Awhsj5CtZWDYwX/aA5RilNcE0Nw/WtKln/ylpnxG3GrBWPOCUxGcIEG4yT/6+pid7Q9ggQQVItOToDCYgJo3qvtnfPRqWTwRBoWon6B0d84RE0d+DWhSWl54SrOGV2KyZoXaKVxYZOkVONMl0qUc1rU789Nu/uALQErzDYbb0Z2MPxUmrwVRQV1NLTdbIlJaaRLSFMoL6r1H4SAkkNa43HVarK42kyaOi8LAa0qWgc0hb3ejUsw2IQCPEJ+FlQzv+j1iDwzbZm7BnIH3NK3+ix7J7usYemePuWLWvUjschFV8qL8kMU0hFLwMwSFawE8FFJqSSLmsp8B5oMsfUSiIvNvLKaxENjMOFI0WG4nCQGL5ITpS/5bBEAqY="
+
+branches:
+  only:
+    - main
+    - /^release-[0-9]+\..*$/
+
+env:
+  global:
+    # Required
+    - OS=linux
+    - COMPONENT_TAG_EXTENSION="-${TRAVIS_COMMIT}"
+
+    # Component Specific
+    - COMPONENT_DEPLOY_COMMAND=${TRAVIS_BUILD_DIR}/build/deploy-to-cluster.sh
+    - COMPONENT_INIT_COMMAND=${TRAVIS_BUILD_DIR}/build/install-dependencies.sh
+    - COMPONENT_BUILD_COMMAND=${TRAVIS_BUILD_DIR}/build/build.sh
+    - COMPONENT_UNIT_TEST_COMMAND=${TRAVIS_BUILD_DIR}/build/run-unit-tests.sh
+    - COMPONENT_E2E_TEST_COMMAND=${TRAVIS_BUILD_DIR}/build/run-e2e-tests.sh
+    - GO111MODULE=on
+
+stages:
+  - build
+  - unit-test
+  - test-e2e
+  - release-ff
+  - publish
+
+before_script:
+  # Bootstrap the build harness.
+  - make
+  # Set the image tag differently for PRs.
+  - if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then export COMPONENT_TAG_EXTENSION="-PR${TRAVIS_PULL_REQUEST}-${TRAVIS_COMMIT}"; fi;
+  # Use ssh to pull from private repos.
+  - git config --global url."git@github.com:".insteadOf "https://github.com/"
```

Comment:
Remove. This breaks release-ff.  I just had to make this change in the collector and aggregator projects.
```suggestion
```

Reply:
I'm committing this minor change.